src/service/viewport/viewport-impl.js

```diff
@@ -999,25 +1013,82 @@ export class Viewport {
     }
   }
 
+  /**
+   * Specifies a callback for measuring an element's fixed offset when the
+   * viewport's padding top changes.
+   *
+   * `measure` runs in a measure phase and takes:
+   *  - afterAnimation
+   *    a promise that resolves after a running translation is finished.
+   *  - lastPaddingTop in px.
+   *  - paddingTop in px.
+   *
+   * And returns a vertical offset at start of animation. End will always be 0.
+   *
+   * This tears down the element from the fixed layer (but does NOT remove from
+   * the transfer layer), so its offset needs to be managed independently
+   * afterwards.
+   *
+   * @param {!Element} element
+   * @param {!FixedElementMeasureFnDef} measure
+   */
+  setFixedElementMeasurer(element, measure) {
+    this.removeFromFixedLayer(
+        element, /* onlyTearDown */ true, /* keepOffset */ true);
```

Comment:
They mean the same thing?

Reply:
They don't? Teardown removes tracking. Keep offset lets the element keep its `top` when tracking is removed. They don't equate logically or semantically.

Reply:
"onlyTearDown" implies "keepOffset", no? That, and the only caller that uses these parameters passes true for both.